piksi_tools/console/settings_view.py

```diff
@@ -212,6 +227,40 @@ def _factory_default_button_fired(self):
                         + "Are you sure you want to reset to factory defaults?"
     confirm_prompt.run(block=False)
 
+  def _settings_save_to_file_fired(self):
```

Comment:
We should leave this settings to be accessible for the user, it is more annoying if you cannot do something related to your device.  I'll remove both save to flash and refresh and make the UI be in sync with flash at every change and then we can see how it looks.